File: Mod3/Clase09/app/src/main/java/com/area51/clase09/database/SQLiteManage.java
package com.area51.clase09.database;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import androidx.annotation.Nullable;

public class SQLiteManage extends SQLiteOpenHelper {
    public SQLiteManage(@Nullable Context context) {
        super(context, "clase", null, 1);
    }

    @Override
    public void onCreate(SQLiteDatabase sqLiteDatabase) {
        sqLiteDatabase.execSQL(
                "create table ubicaciones(" +
                        "id integer primary key autoincrement," +
                        "direccion text," +
                        "latitud decimal(10,7)," +
                        "longitud decimal(10,7))"
        );
    }

    @Override
    public void onUpgrade(SQLiteDatabase sqLiteDatabase, int i, int i1) {

    }
}

File: Mod3/Clase09_1/app/src/main/java/com/area51/clase09_1/database/SQLiteManage.kt
package com.area51.clase09_1.database

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

class SQLiteManage(context: Context) :
    SQLiteOpenHelper(context, "clase", null, 1) {

    override fun onCreate(p0: SQLiteDatabase?) {
        p0!!.execSQL(
            "create table ubicaciones(" +
                    "id integer primary key autoincrement," +
                    "direccion text," +
                    "latitud decimal(17,2)," +
                    "longitud decimal(17,2))"
        )
    }

    override fun onUpgrade(p0: SQLiteDatabase?, p1: Int, p2: Int) {

    }
}